docs/content/singleton-actors.md

```diff
@@ -18,44 +18,23 @@ Due to Unreal server-workers spawning their own instances of each Singleton Acto
 ## Setting up Singleton Actors
 
 To properly set up Singleton Actors for your project, you need to:
-1. Register Singleton Actors with the `DefaultEditorSpatialGDK.ini` file
+1. Register Singleton Actors by tagging them with the `SpatialType=Singleton` class attribute.
 1. Add the generated components to the UnrealWorker worker configuration file
 
 ### Registering Singleton Actors
```

Comment:
These headers are a bit odd now. 
Do we need the title "### Registering Singleton Actors"? 
Can it be "##How to tag classes with Singleton Actor identifiers" <-Note 2 * # for sub-header.